mock/src/components/mockedJson.ts

```diff
@@ -0,0 +1,13 @@
+export let datamocked: {
+  [key: string]: string[][];
+} = {};
+
+datamocked["strings.csv"] = [
+  ["The", "song", "remains", "the", "same."],
+  ["The", "song", "remains", "the", "same."],
+];
+
+datamocked["more_strings.csv"] = [
+  ["More", "strings", "for"],
+  ["the", "string", "god"],
+];
```

Comment:
We suggest to build different shapes of mocked datas (i.e. empty data, data with just one column, or data with just one row). This may help test expected HTML table outputs and ensure that the visual for those datas are delivered as expected in the front-end.